products/bpmsuite/_common/product.yml

```diff
@@ -5,9 +5,9 @@ has_installer: true
 current_version: 6.1.0.GA
 documentation_minor_version: 6.1
 xpaas_url: https://www.openshift.com/developers/jboss-bpms
-intro_video: https://vimeo.com/95505857
-vimeo_album: http://vimeo.com/album/2982446
-youtube_album: https://www.youtube.com/playlist?list=PLf3vm0UK6HKphPQWLX8SPJlPyqDhpZS4U
+intro_video: https://vimeo.com/142667410
```

Comment:
So did you pull the video (somehow) from Red Hat's Vimeo account and put it on your own? In that case, how about you do the same, but put it onto JBoss Developer's Vimeo account?

Also, if you pass me a link to the original video on Red Hat's Vimeo account I can try and see why the site wouldn't accept it.